src/css/imports/title.less

```diff
@@ -48,7 +47,5 @@
 
   .jw-title-secondary {
     display: none;
-    margin-top: 5px;
```

Comment:
This also removes the padding above item descriptions in the idle state. you're now only setting it in the error state.